Source/HYPFormsCollectionViewDataSource.m

```diff
@@ -451,6 +466,13 @@ - (void)fieldCell:(UICollectionViewCell *)fieldCell updatedWithField:(HYPFormFie
     }
 }
 
+- (void)fieldCell:(UICollectionViewCell *)fieldCell processTargets:(NSArray *)targets
+{
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.2 * NSEC_PER_SEC)), dispatch_get_main_queue(), ^{
```

Comment:
In a previous change you used 0.1 seconds for the delay, now you use 0.2 seconds. Is that intentional? Shouldn't it be the same all the time? Maybe defined by a constant?